vllm/model_executor/models/qwen2_5_vl.py

```diff
@@ -428,6 +428,14 @@ def forward(
             )
         elif self.attn_backend == _Backend.TORCH_SDPA:
             # Execute attention entry by entry for speed & less VRAM.
+            from vllm.platforms import current_platform
+
+            # Never remove the next contiguous logic
+            # Without it, hallucinations occur with the backend
+            if current_platform.is_rocm():
```

Comment:
Ah I was wondering why I didn't observe this in testing on my end - I didn't test on rocm :( Sorry for missing this, and thanks for documenting in code so we avoid easy to miss mistakes like this in the future!